src/stacking-group.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import {useIsInverseVariant} from './theme-variant-context';
+import {vars} from './skins/skin-contract.css';
+import Inline from './inline';
+
+type Props = {
+    stacked?: boolean;
+    moreItemsStyle: {
+        type: 'circle' | 'square';
+        size: number;
+    };
+    maxItems?: number;
+    children: React.ReactNode;
+};
+
+const StackingGroup: React.FC<Props> = ({moreItemsStyle, stacked = false, maxItems = Infinity, children}) => {
+    const isInverse = useIsInverseVariant();
+    const backgroundColor = isInverse ? vars.colors.brandHigh : vars.colors.brandLow;
+    const textColor = isInverse ? vars.colors.textPrimaryInverse : vars.colors.brand;
+    const border = stacked ? `1px solid ${vars.colors.borderLow}` : 'none';
+
+    const styleCircle = moreItemsStyle.type === 'circle';
+    const space = stacked ? -8 : 8;
+    const countChildren = React.Children.count(children);
+    const plusCase = countChildren - maxItems;
+
+    const styleBorderRadius = styleCircle ? vars.borderRadii.avatar : vars.borderRadii.container;
+    const size = moreItemsStyle.size;
+
+    return (
+        <Inline space={space}>
+            {React.Children.map(children, (child, index: number) => index < maxItems && <>{child}</>)}
+            {countChildren > maxItems && (
+                <div>
+                    <div
+                        style={{
+                            position: 'relative',
+                            width: size,
+                            height: size,
+                            backgroundColor,
+                            borderRadius: styleBorderRadius,
+                            boxSizing: 'border-box',
+                            border,
+                            zIndex: 1,
+                        }}
+                    >
+                        <div
+                            style={{
+                                display: 'flex',
+                                position: 'absolute',
+                                width: size,
+                                height: size,
+                                justifyContent: 'center',
+                                alignItems: 'center',
+                                color: textColor,
+                            }}
+                            aria-label={'+' + plusCase.toString()}
+                        >
+                            {'+' + plusCase}
```

Comment:
sorry for opening a review again, I didn't see this detail. Can you use `<Text4>` for the text inside more item element?

This information wasn't in the specs, but all the text should use a text preset. I add it to specs
![image](https://github.com/Telefonica/mistica-web/assets/6722153/c68302a8-96de-45fa-8a13-dce4712f487c)

Reply:
No problem, I already put the change